tests/fixtures/line-break.js

```diff
@@ -0,0 +1,29 @@
+/** @jsx h */
+
+import h from '../h';
+
+const input = (
+    <value>
+        <document>
+            <paragraph>
+                {
+                    "This is a slate-hyperprint's test for strings with\n\nline breaks."
+                }
+            </paragraph>
+        </document>
+    </value>
+);
+
+const output = `
+<value>
+    <document>
+        <paragraph>
+            {
+                "This is a slate-hyperprint's test for strings with\\n\\nline breaks."
```

Comment:
The output should be a valid input as well. Have you checked that you could write it like this in JSX? If possible, the `input` should be written the exact same way than the `output`

Reply:
Ok it's because in a templating string